Actions/Sign/Sign.psm1

```diff
@@ -0,0 +1,41 @@
+function Get-NavSipFromArtifacts
+(
+    [string] $NavSipDestination
+) 
+{
+    #TODO: It would be nice with a different approach here - This downloads a lot of unnecessary stuff
```

Comment:
Are we going to look for TODOs in the code once in a while? I think not... better create a work item to track this new approach.

Reply:
Making a note of it :)